README.md

```diff
@@ -1,34 +1,120 @@
-# Hello World
-Hello World is a project about Introduction to Microverse
-# Built With
-it is built with Html & CSS
-# Key Features
-Html
-css
```

Comment:
Please consider adding a Key-Features section to your README as presented in the [Microverse README template](https://github.com/microverseinc/readme-template) where you add some of the features of your project. As it's a project for like starter so don't worry and add whatever you think you have done in it like:-

1- Setup Linters.
2- Added professional README
3- Used GitHub flow

![image](https://user-images.githubusercontent.com/94223367/219093883-ff08c574-f60d-43ce-91ef-c3cc3227f2c1.png)